src/wallet/test/wallet_tests.cpp

```diff
@@ -137,10 +140,10 @@ BOOST_FIXTURE_TEST_CASE(scan_for_wallet_transactions, TestChain100Setup)
 
     // Prune the older block file.
     {
-        LOCK(cs_main);
+        LOCK(::cs_main);
         Assert(m_node.chainman)->m_blockman.PruneOneBlockFile(oldTip->GetBlockPos().nFile);
+        UnlinkPrunedFiles({oldTip->GetBlockPos().nFile});
     }
-    UnlinkPrunedFiles({oldTip->GetBlockPos().nFile});
```

Comment:
0bfa32b131ed2da8799e1d1239bf390dd612b7bb
`UnlinkPrunedFiles` includes a logging operation. Could we keep logging out of the `::cs_main` lock?

Reply:
done